src/lib/Navbar.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import { page as currentPage } from '$app/stores';
+	import { mdiMenu, mdiClose } from '@mdi/js';
+
+	export let logoUrl: string;
+	export let siteName: string;
+	type Page = { name: string; url: string };
+	export let pages: readonly Page[];
+
+	let menuExpanded = false;
+</script>
+
+<header>
+	<div class="container">
+		<nav>
+			<ul>
+				<li>
+					<a href="/" class="brand contrast"><img src={logoUrl} alt="" />{siteName}</a>
+				</li>
+			</ul>
+			<ul>
+				<li>
+					<button
+						aria-label={menuExpanded ? 'Close' : 'Menu'}
+						on:click={() => (menuExpanded = !menuExpanded)}
+					>
+						<svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" viewBox="0 0 24 24">
+							<path fill="currentColor" d={menuExpanded ? mdiClose : mdiMenu} />
+						</svg>
+					</button>
+				</li>
+			</ul>
+			<ul class="menu" class:expanded={menuExpanded}>
+				{#each pages as page}
+					<li>
+						<a
+							href={page.url}
+							on:click={() => (menuExpanded = false)}
+							aria-current={$currentPage.url.pathname == page.url ? 'page' : undefined}
+						>
+							{page.name}
+						</a>
+					</li>
+				{/each}
+			</ul>
+		</nav>
+	</div>
+</header>
+
+<style>
+	header {
+		position: sticky;
+		z-index: 2;
+		top: 0;
+		left: 0;
+		right: 0;
+		background-color: color-mix(in lab, var(--pico-background-color) 90%, transparent);
+		backdrop-filter: blur(12px);
+		box-shadow:
+			0 3px 6px rgba(0, 0, 0, 0.16),
+			0 3px 6px rgba(0, 0, 0, 0.23);
+	}
+
+	.brand {
+		display: grid;
+		grid-template-columns: auto auto;
+		align-items: center;
+		gap: 8px;
+		font-size: 1.5em;
+		font-weight: 700;
+
+		& > img {
+			height: 1lh;
+		}
+	}
+
+	@media (width >= 1024px) {
+		ul:has(button) {
+			display: none;
+		}
+	}
+
+	@media (width < 1024px) {
+		li:has(button) {
+			padding: 0;
+		}
+
+		button {
+			appearance: none;
+			background: none;
+			border: none;
```

Comment:
Buttons have blue text by default on iOS
![IMG_0397](https://github.com/kendoclub-at-umich/website/assets/44071655/f7b5535a-05a4-4965-8a74-1baee00bdf0b)
You can fix this with `color: inherit`.

Reply:
fixed in [ddc6a1d](https://github.com/kendoclub-at-umich/website/pull/40/commits/ddc6a1de116a629025ac7d03d2b6d7732ee857df)

Reply:
Looks good
![IMG_0398](https://github.com/kendoclub-at-umich/website/assets/44071655/d96b653d-8480-487e-b9bc-698a614450da)